docs/superpowers/plans/2026-04-01-global-extension-bugs.md

```
@@ -0,0 +1,369 @@
+from zndraw.exceptions import (
+    InternalServerError,
+    ProblemError,
+    UnprocessableContent,
+    problem_exception_handler,
+)
+```
+
+- [ ] **Step 3: Refactor `_validation_exception_handler` to delegate**
+
+Replace the existing `_validation_exception_handler` (lines 58-71):
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)